gitbot/lib.py

```diff
@@ -169,8 +169,16 @@ def bump_version(
         run(f"git config user.name {COMMITTER_NAME}", cwd=repo_root)
         run(f"git config user.email {COMMITTER_EMAIL}", cwd=repo_root)
 
-        command = bump_command(ref_sha, author)
-        run(command, cwd=repo_root)
+        try:
+            command = bump_command(ref_sha, author)
+            run(command, cwd=repo_root)
+        except CommandError as e:
+            execution = run("git show", cwd=repo_root)
+            # e.g. https://github.com/getsentry/getsentry/pull/7672/commits
+            if execution.stdout.find(f"getsentry/sentry@{ref_sha}") > -1:
+                logger.info("The developer has manually bumped the version.")
+            else:
+                raise e
```

Comment:
reraise in python should just be `raise` with no arguments: https://youtu.be/yj4z1N3_0sw